### PR TITLE
Taqccdxxxv 45

### PR DESCRIPTION
**Pull request by ollehhh**

### CODE REVIEW

File: src/main/java/entity/Product.java [hunk 1]
```diff
@@ -103,4 +105,32 @@ public String getDimensions() {
...
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Product product = (Product) o;
+        return id.equals(product.id) &&
+                productName.contains(product.productName) &&
+                prise.equals(product.prise) &&
+                summary.contains(product.summary);
+    }
```
Comment (OrestZhmurkevych) [inline src/main/java/entity/Product.java:130]: It is better practice to use one return statement in the method body. You can fix it with a local boolean variable called "result" or something like that

File: src/main/java/pageobjects/ProductUnitPageObject.java [hunk 2]
```diff
@@ -45,19 +43,11 @@ public String getDescriptionOfProduct() {
...
-        return priceOld.getText();
-    }
-
-    public String getExTax() {
-        exTax = new Label(elementToParse, ProductUnitLocators.EX_TAX_LOC);
-        return exTax.getText();
+    public String getPrice() {
+        priceOfProduct = new Label(elementToParse, ProductUnitLocators.NEW_PRICE_LOC);
+        String price = priceOfProduct.getText();
+        price = price.replaceAll("Ex Tax:", "").replaceAll(",", "");
```
Comment (OrestZhmurkevych) [inline src/main/java/pageobjects/ProductUnitPageObject.java]: You can merge rows 48 and 49 in just one

File: src/main/java/pageobjects/ProductUnitPageObject.java [hunk 3]
```diff
@@ -45,19 +43,11 @@ public String getDescriptionOfProduct() {
...
+        return price.substring(1, price.indexOf(" "));
```
Comment (OrestZhmurkevych) [inline src/main/java/pageobjects/ProductUnitPageObject.java:49]: Also you can merge all this in one row with return statement without even using String variable

**Review by OrestZhmurkevych — COMMENTED** [review]
Fix pls that small style and design issues

**Review by YuraSmoliy — APPROVED** [review]

**Review by OrysiaBen — APPROVED** [review]

**Review by YuraSmoliy — APPROVED** [review]

**Review by NataliaHardzhalo — APPROVED** [review]

**Pull request merged**